VP_VKD3D_requirements.json

```diff
@@ -0,0 +1,759 @@
+                }
+            }
+        },
+        "d3d12_level_11_0_mutable_descriptor_type_valve": {
+            "extensions": {
+                "VK_VALVE_mutable_descriptor_type": 1
+            },
+            "features": {
+                "VkPhysicalDeviceMutableDescriptorTypeFeaturesVALVE": {
+                    "mutableDescriptorType": true   
+                }
+            }
+        },
+        "d3d12_level_11_1": {
+            "extensions": {
+            },
+            "features": {
+                "VkPhysicalDeviceFeatures": {
+                    "logicOp": true,
+                    "vertexPipelineStoresAndAtomics": true
+                }
+            },
+            "properties": {
+                "VkPhysicalDeviceProperties": {
+                    "limits": {
+                        "maxPerStageDescriptorStorageBuffers": 64,
+                        "maxPerStageDescriptorStorageImages": 64
+                    }
+                }
+            }
+        },
+        "d3d12_level_12_0": {
+            "extensions": {
+                "VK_EXT_sampler_filter_minmax": 1
+            },
+            "features": {
+                "VkPhysicalDeviceFeatures": {
+                    "sparseBinding": true,
+                    "sparseResidencyAliased": true,
+                    "sparseResidencyBuffer": true,
+                    "sparseResidencyImage2D": true,
+                    "shaderResourceResidency": true,
+                    "shaderResourceMinLod": true
+                }
+            },
+            "properties": {
+                "VkPhysicalDeviceProperties": {
+                    "sparseProperties": {
+                        "residencyStandard2DBlockShape": true,
+                        "residencyNonResidentStrict": true,
+                        "residencyAlignedMipSize": false
+                    }
+                },
+                "VkPhysicalDeviceVulkan11Properties": {
+                    "subgroupSize": 4,
+                    "subgroupSupportedOperations": [ "VK_SUBGROUP_FEATURE_ARITHMETIC_BIT", "VK_SUBGROUP_FEATURE_BASIC_BIT", "VK_SUBGROUP_FEATURE_BALLOT_BIT", "VK_SUBGROUP_FEATURE_SHUFFLE_BIT", "VK_SUBGROUP_FEATURE_QUAD_BIT", "VK_SUBGROUP_FEATURE_VOTE_BIT" ],
+                    "subgroupSupportedStages": [ "VK_SHADER_STAGE_COMPUTE_BIT", "VK_SHADER_STAGE_FRAGMENT_BIT" ]
+                },
+                "VkPhysicalDeviceVulkan12Properties": {
+                    "filterMinmaxSingleComponentFormats": true
+                }
+            },
+            "queueFamiliesProperties": [
+                {
+                    "VkQueueFamilyProperties": {
+                        "queueFlags": [ "VK_QUEUE_SPARSE_BINDING_BIT" ],
+                        "queueCount": 1
+                    }
+                }
+            ]
+        },
+        "d3d12_level_12_1": {
+            "extensions": {
+                "VK_EXT_conservative_rasterization": 1
```

Comment:
Missing VK_EXT_fragment_shader_interlock with per-pixel and per-sample features.

Reply:
Fixed